src/relay/backend/te_compiler.cc

```diff
@@ -527,8 +526,8 @@ class TECompilerImpl : public TECompilerNode {
 
   /*! \brief compiler cache lock*/
   std::mutex mutex_;
-  /*! \brief internal name map to get an unique name */
-  std::unordered_map<std::string, int> name_map_;
+  /*! \brief internal GlobalVarSupply to get unique GlobalVars  */
+  GlobalVarSupply global_var_supply = GlobalVarSupply();
```

Comment:
Please just leave it as `GlobalVarSupply global_var_supply;`.  Also, add `_` to the name (as with other members).